### PR TITLE
FS2-2454: Small multiples reset button and fit lines

### PR DESCRIPTION
**Pull request by chschan**

### CODE REVIEW

File: theSrc/test/snapshots/travis/master/testPlans/bubbleplots/bubbleplot_small_multiples-snap.png
Comment (JustinCCYap) [inline theSrc/test/snapshots/travis/master/testPlans/bubbleplots/bubbleplot_small_multiples-snap.png:1]: Why did the gap between the panels become 0?

File: theSrc/test/snapshots/travis/master/testPlans/functionalTest_old/smallmultiples_after_reset-snap.png
Comment (JustinCCYap) [inline theSrc/test/snapshots/travis/master/testPlans/functionalTest_old/smallmultiples_after_reset-snap.png:1]: Should the reset button still appear after reset?

File: theSrc/test/snapshots/travis/master/testPlans/bubbleplots/bubbleplot_small_multiples-snap.png
Comment (chschan) [reply theSrc/test/snapshots/travis/master/testPlans/bubbleplots/bubbleplot_small_multiples-snap.png:1]: I changed the config to test the panelXGap parameter: https://github.com/Displayr/rhtmlCombinedScatter/pull/27/files#diff-7d9aa55da40511c9742aa9c0cb5b7074a6139af4bb7eed627cc9d4d9708b5ac3

**Review by JustinCCYap — APPROVED** [review]

File: theSrc/test/snapshots/travis/master/testPlans/functionalTest_old/smallmultiples_after_reset-snap.png
Comment (chschan) [reply theSrc/test/snapshots/travis/master/testPlans/functionalTest_old/smallmultiples_after_reset-snap.png:1]: Thanks for pointing that out. I've updated the test to move the mouse away and checks that the reset button is hidden

**Pull request merged**